bitrise.yml

```diff
@@ -131,6 +131,8 @@ workflows:
             Commit | ${GIT_CLONE_COMMIT_MESSAGE_SUBJECT}
             Source-Workflow | ${BITRISE_TRIGGERED_WORKFLOW_TITLE}
             Branch | ${BITRISE_GIT_BRANCH}
+            Warnings-in-code | ${SHOW_WARNING_COUNT}
```

Comment:
As we have talked, wee need two counts as the warnings shown when building + unit tests are not all. There are more, and different when running build + xcuitest